docs/database.md

```diff
@@ -281,7 +281,7 @@ from alembic.config import Config
 
 @pytest.fixture(scope="session", autouse=True)
 def create_test_database():
-    url = str(app.DATABASE_URL)
+    url = str(app.database._url)
```

Comment:
`app.DATABASE_URL` is already correct in the docs here, but we should add `import app` above.
(It's *not* the application instance, it's the module)

Reply:
Please verify tom. My issue was that the next assert:
`assert not database_exists(url)` was failing because the value of `app.DATABASE_URL` was the dev value, not the TEST_DATABASE_VALUE.

According to the docs, the `database` gets generated this way:
```
if TESTING:
    database = databases.Database(TEST_DATABASE_URL, force_rollback=True)
else:
    database = databases.Database(DATABASE_URL)
```

so the assert can only check against `database._url`. Or at least that's what I understood.

Maybe I've missed something hehe

Reply:
I mean, not matter what I did, even if `TESTING` is `True`, the assert would always fail.

Reply:
Right, then `app.TEST_DATABASE_URL`

Reply:
But isn't the idea to check that you are not referencing the "prod/dev" database? if you point it to `TEST_DATABASE_URL` it won't fail.